kv/key.go

```diff
@@ -158,9 +159,15 @@ type Handle interface {
 	// String implements the fmt.Stringer interface.
 	String() string
 	// MemUsage returns the memory usage of a handle.
-	MemUsage() int64
+	MemUsage() uint64
+	// ExtraMemSize returns the dynamic size of memory occupied by the handle, e.g. slices.
```

Comment:
`dynamic` means the part of memory is not a fixed number, i.e. slices or pointers. Do you have any suggestions on improving the comment?
We use it with the mem-aware map, since the map only counts the actual size of the keys and values, but doesn't contain the objects that are referenced by pointers.